lib/utils/import.js

```diff
@@ -9,11 +9,59 @@ const {
 } = require('../utils/types');
 
 module.exports = {
+  gatherImports,
+  getImportFromMap,
   getSourceModuleNameForIdentifier,
   getSourceModuleName,
   getImportIdentifier,
 };
 
+const IMPORTS_TO_GATHER = [
+  ['ember'],
+  ['@ember/object', 'computed'],
+  ['@ember/object', 'get'],
+  ['@ember/object', 'getProperties'],
+  ['@ember/object', 'getWithDefault'],
+];
+
+/**
+ * Gathers the names that common imports are imported under.
+ * @param {map<string,string>} imports map of each imported path/function to the name its imported under
+ * @returns {function} a visitor for ImportDeclaration nodes
+ */
+function gatherImports(imports) {
+  return function (node) {
+    IMPORTS_TO_GATHER.forEach((importToGather) => {
+      const [path, namedImport] = importToGather;
+      const key = getImportMapKey(path, namedImport);
+      if (node.source.value === path) {
+        imports.set(key, imports.get(key) || getImportIdentifier(node, path, namedImport));
+      }
+    });
+  };
+}
+
+/**
+ * Generates a key to use for the imports map.
+ * @param {string} path
+ * @param {string} namedImport
+ * @returns {string} key to use for map
+ */
+function getImportMapKey(path, namedImport) {
+  return `${path}|${namedImport}`;
+}
+
+/**
+ * Gets the name that the path/function is imported under from the map.
+ * @param {map<string,string>} map
+ * @param {string} path
+ * @param {string} namedImport
```

Comment:
We should document that `namedImport` is optional. I think we should also make the key just `path` in that case, instead of `path|`.